fdbserver/workloads/GetMappedRange.actor.cpp

```diff
@@ -29,11 +29,23 @@
 #include "flow/flow.h"
 #include "flow/actorcompiler.h" // This must be the last #include.
 
+using Result = std::variant<MappedRangeResult, MappedRangeResultV2>;
+
 const Value EMPTY = Tuple().pack();
 ValueRef SOMETHING = "SOMETHING"_sr;
 const KeyRef prefix = "prefix"_sr;
 const KeyRef RECORD = "RECORD"_sr;
 const KeyRef INDEX = "INDEX"_sr;
+const int MATCH_INDEX_TEST_710_API = -1;
+const int code_int = 1;
+const int code_bool = 2;
+const int code_invalid = 0;
+const int VERSION = 2;
+const int INVALID_VERSION = 1;
+static std::unordered_map<int, int> versionToPosMatchIndex = { std::make_pair(2, 1) };
+static std::unordered_map<int, int> versionToPosFetchLocalOnly = { std::make_pair(2, 6) };
+static std::unordered_map<int, int> versionToPosLocal = { std::make_pair(2, 1) };
+static std::unordered_map<int, int> versionToLength = { std::make_pair(2, 8) };
```

Comment:
These seem to appear in "storageserver.actor.cpp" as well. So it's probably better to declar them in a separate header file.